net/DevExtreme.AspNet.Data.Tests/GroupHelperTests.cs

```diff
@@ -127,6 +127,31 @@ public class GroupHelperTests {
         }
 
         [Fact]
+        public void GroupWithNullValues() {
```

Comment:
Rename to`GroupInterval_NullDates` to be consistent with the existing test case and to denote that this case relates to group intervals